.gitignore

```diff
@@ -2,5 +2,7 @@ node_modules
 .browser_modules
 lib
 *.log
-*-app/*
-!*-app/package.json
+example/*-app/*
+!example/*-app/package.json
+
+example/modelsever-example/build/com\.eclipsesource\.modelserver\.example-0\.0\.1-SNAPSHOT-standalone\.jar
```

Comment:
The folder name has a type in it, it should read modelse**r**ver-example.

Reply:
> Wrong folder name. If I follow 'Developing with the Browser example' section in the README I also run into issues with `Failed to resolve module: modelserver-example`. Probably related to the wrong folder name?

It works for me so I don't think this is related to the wrong folder name. Could you execute "yarn" on the root level and then test again?

Btw. the readme is autogenerated from the yo-generator for Theia-extensions and could potentially be outdated.

Reply:
I see, but I suppose we can just update it? Running `yarn` on the root level works, thanks.